src/pages/documentation/adobe-target/release-notes.md

```diff
@@ -1,5 +1,11 @@
 # Release Notes
 
+## March 2, 2023
+
+### iOS Target 3.3.1
+
+* `target.previewEnabled` configuration setting is no longer required to enable preview mode in Target iOS SDK. If not configured, the default value for the setting will be assumed to be true.
```

Comment:
I think we can say it's set to true, rather than assumed to be true? Or am I misunderstanding what's going on?

Reply:
sure, yeah I meant the same that we consider default value as true. I used assume to indicate that it's not explicitly set by the user but I. suppose it's misleading. Let me rephrase it. Ty